### ISSUE TITLE
NTR, pulmo: Hypertrophic alveolar type II cells

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#6387]
**Preferred term label:**
Hypertrophic alveolar type II cells

**Synonyms**
Swelling of type II pneumocytes

**Definition (free text, please give PubMed ID)**
PMID: 2166657

**Parent term (use hpo.jax.org/app)**
NEW Parent term: Abnormality of alveolar type II pneumocytes

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
DIP OMIM:  263000

**Your nano-attribution (ORCID)**
LMU:chrapp, @kknoflach, @mgriese

**Comment by pnrobinson** [comment]
I do not think we should use 'swelling' in the definition, because it sounds like edema (more water). 
I think the generic definition for hypertropy is more like this
def:Increase in size of type II pneumocytes.

**Comment by pnrobinson** [comment]
This is https://hpo.jax.org/app/browse/term/HP:0033246
